website/screens/components/date-input/code/DateInputCodePage.tsx

```diff
@@ -0,0 +1,239 @@
+import { DxcStack, DxcTable, DxcText } from "@dxc-technology/halstack-react";
+import QuickNavContainer from "@/common/QuickNavContainer";
+import QuickNavContainerLayout from "@/common/QuickNavContainerLayout";
+import DocFooter from "@/common/DocFooter";
+import Code from "@/common/Code";
+import Example from "@/common/example/Example";
+import basic from "./examples/basic";
+import optional from "./examples/optional";
```

Comment:
Remove unused import which is causing problems.

Reply:
done